<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>canvas_layer_container</class>
 <widget class="QWidget" name="canvas_layer_container">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="layer_list">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="layer_list_layout">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>380</width>
        <height>248</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2"/>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="add_layer">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Add new layer</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../../../../gui/resources/feather.qrc">
         <normaloff>:/dark/icons/feather/dark/plus.svg</normaloff>:/dark/icons/feather/dark/plus.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="move_layer_up">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Move selected layer up</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../../../../gui/resources/feather.qrc">
         <normaloff>:/dark/icons/feather/dark/chevron-up.svg</normaloff>:/dark/icons/feather/dark/chevron-up.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="move_layer_down">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Move selected layer down</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../../../../gui/resources/feather.qrc">
         <normaloff>:/dark/icons/feather/dark/chevron-down.svg</normaloff>:/dark/icons/feather/dark/chevron-down.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_layer">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Delete selected layer</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../../../../gui/resources/feather.qrc">
         <normaloff>:/dark/icons/feather/dark/trash-2.svg</normaloff>:/dark/icons/feather/dark/trash-2.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../../gui/resources/feather.qrc"/>
 </resources>
 <connections/>
</ui>
